wasm/src/programs/program.rs

```diff
@@ -313,6 +313,15 @@ impl Program {
         }
 
         Reflect::set(&input, &"members".into(), &record_members).map_err(|_| "Failed to set property")?;
+
+        // Adding _nonce object to record
+        let _nonce = Object::new();
+        Reflect::set(&_nonce, &"name".into(), &"_nonce".into()).map_err(|_| "Failed to set property")?;
+        Reflect::set(&_nonce, &"type".into(), &"group".into()).map_err(|_| "Failed to set property")?;
+        Reflect::set(&_nonce, &"visibility".into(), &"public".into()).map_err(|_| "Failed to set property")?;
+
+        Reflect::set(&input, &"_nonce".into(), &_nonce).map_err(|_| "Failed to set property")?;
+
```

Comment:
Per our discussion offline I think that you're right it would probably make more sense for this to be part of the members